services/ui-src/src/utils/forms/forms.ts

```diff
@@ -228,3 +231,80 @@ export const resetClearProp = (fields: (FormField | FormLayoutElement)[]) => {
     }
   });
 };
+
+export const convertEntityToTargetPopulationChoice = (
+  entity: EntityShape[]
+) => {
+  return entity?.map((field: EntityShape) => {
+    return {
+      key: `targetPopulations-${field.id}`,
+      value: field.isRequired
+        ? field.transitionBenchmarks_targetPopulationName
+        : `Other: ${field.transitionBenchmarks_targetPopulationName}`,
+    };
+  });
+};
+
+export const convertChoiceToEntity = (choices: Choice[]) => {
+  return choices?.map((field: Choice) => {
+    return {
+      id: field.key,
+      label: field.value,
+      name: field.value,
+      value: field.value,
+    };
+  });
+};
+
+export const convertTargetPopulationsFromWPToSAREntity = (
+  targetPopulations: any[]
```

Comment:
does this have to be an any? can it be AnyObject[]?

Reply:
Yes!